doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -1157,3 +1157,8 @@ result in undefined behavior.  It now throws a `ValueError`.
 
 The signature of the (private) ``Axis._update_ticks`` has been changed to not
 take the renderer as argument anymore (that argument is unused).
+
+Reduced default value of `axes.formatter.limits`
```

Comment:
Sorry, should have been clear about where this goes.  it gets a new file in `doc/api/next-api-changes/`

Reply:
@jklymak I am not able to exactly understand the naming convention of the files in `doc/api/next-api-changes/`. For eg, in `2019-05-08-AL.rst` what exactly does AL mean?

Reply:
`AL` are the authors initials.  Which is a bit mysterious, i.e. in that case "Antony Lee".

Reply:
And the date is the date of modification? Right?

Reply:
Yes